### PR TITLE
K3S/Calico Implementation tutorials

### PR DESCRIPTION
**Pull request by frozenprocess**
## Description
How to implement K3S and with Calico.

## Todos
- [x] structure (M)
- [ ] etcd ** omitted **
- [ ] calicoctl ** omitted **
- [x] Detailed tutorials
- [ ] Flannel and Calico tutorial ** omitted **
- [ ] ARM based tutorial (M) ** omitted **
- [ ] Tests ** omitted **
- [x] Documentation

** Stuff marked with `(M)` should be consulted with maintainers.**
## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 5713fbf80332fc46b4c7bd8d91844c6e6c2718b3

https://deploy-preview-3582--calico.netlify.app

**Comment by frozenprocess** [comment]
@hogepodge, @lxpollitt  I created a subset of links in the `install calico` section of sidebar menu.
Is this the right approach or should it be in different place?

I have an agenda for the tutorials to come in TODO section, I would be happy if you could sharing anything that might missed.

**Comment by frozenprocess** [comment]
Some of the items from agenda were omitted since it was redundant.

**Comment by frozenprocess** [comment]
@caseydavenport I'm not sure why semaphoreci build fails ... any suggestions?

File: getting-started/kubernetes/k3s/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+### Requirements
+
+- x86-64 processor
+- 1CPU
+- 1GB Ram
+- 10GB free disk space
+- Ubuntu 16.04 (amd64), Ubuntu 18.04 (amd64)
+   > **Note**: K3S supports ARM proccessors too, this tutorial was tested against x86-64 processor environment.
+   > For more detail please visit [this link](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#operating-systems).
```
Comment (lxpollitt) [inline getting-started/kubernetes/k3s/quickstart.md:21]: Use the magic include macro for external links.

File: getting-started/kubernetes/k3s/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+   It should return something like the following.
+
+   ```
+   NAME         STATUS   ROLES    AGE   VERSION        INTERNAL-IP    EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION       CONTAINER-RUNTIME
+   k3s-master   Ready    master   40m   v1.18.2+k3s1   172.16.2.128   <none>        Ubuntu 18.04.3 LTS   4.15.0-101-generic   containerd://1.3.3-k3s2
+   ```
+   {: .no-select-button}
+
+Congratulations! You now have a single-host K3S cluster
```
Comment (lxpollitt) [inline getting-started/kubernetes/k3s/quickstart.md:121]: single-host -> single node

**Pull request merged**